nixos/modules/services/mail/postal.nix

```
@@ -0,0 +1,530 @@
+{
+  config,
+  lib,
+  options,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.postal;
+  opt = options.services.postal;
+
+  yaml = pkgs.formats.yaml { };
+
+  postalConfigFile = yaml.generate "postal-config.yml" cfg.settings;
+
+  wrapper = pkgs.writeShellScriptBin "postal" (
+    lib.concatStrings [
+      (lib.optionalString (cfg.environmentFile != null) ''
+        export $(cat ${cfg.environmentFile} | xargs)
+      '')
+      ''
+        export POSTAL_CONFIG_FILE_PATH=${postalConfigFile}
+        exec ${pkgs.postal}/bin/postal "$@"
+      ''
+    ]
+  );
+
+  postalCli = pkgs.symlinkJoin {
+    name = "postal";
+    paths = [ wrapper ];
+  };
+
+in
+{
+  options.services.postal = {
+    enable = mkEnableOption "Enable Postal service" // {
```

```suggestion
    enable = mkEnableOption "postal" // {
```

Otherwise, this will come out as "Whether to enable Enable Postal service" in the manual and search